Rakefile

```diff
@@ -1 +1,91 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
+require 'rake/clean'
+require 'cucumber/rake/task'
+require 'mechanize'
+require 'fileutils'
+require 'yaml'
+require 'launchy'
+
+CDK_DOWNLOAD_URL='https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software'
+CDK_BOX_BASE_NAME='rhel-cdk-kubernetes-7.2-23.x86_64.vagrant'
+
+ADB_DOWNLOAD_URL='http://cloud.centos.org/centos/7/atomic/images'
+ADB_BOX_BASE_NAME='AtomicDeveloperBundle-2.0.0-CentOS7'
+
+CLOBBER.include('pkg')
+CLEAN.include('build')
+
+Cucumber::Rake::Task.new(:features)
+
+namespace :features do
+  cucumber_config = YAML.load_file('./.config/cucumber.yml')
+  cucumber_config.each_key do |key|
+    next if key == 'default'
+    Cucumber::Rake::Task.new(key.to_sym, "Run Cucumber features using profile '#{key}' (see cucumber.yml)'") do |t|
+      t.profile = key
+    end
+  end
+
+  desc 'Opens the HTML Cucumber test report'
+  task :open_report do
+    Launchy.open('./build/features_report.html')
+  end
+end
+
+desc 'Download CDK Vagrant box using the specified provider (default \'virtualbox\')'
+task :get_cdk, [:provider] do |t, args|
+  provider = args[:provider].nil? ? 'virtualbox' : args[:provider]
+  agent = Mechanize.new
+  agent.follow_meta_refresh = true
+  agent.get(CDK_DOWNLOAD_URL) do |page|
+
+    # Submit first form which is the redirect to login page form
+    login_page = page.forms.first.submit
+
+    # Submit the login form
+    after_login = login_page.form_with(:name => 'login_form') do |f|
+     username_field = f.field_with(:id => 'username')
+     username_field.value = 'service-manager@mailinator.com'
```

Comment:
Right. Avoids to pass anything in via environment variables. Works out of the box.